docs/modules/user-guide/partials/proc_adding-event-bindings.adoc

```diff
@@ -50,11 +42,5 @@ events:
 +
 [role="_additional-resources"]
 .Additional resources
-<<<<<<< HEAD
-* See https://github.com/che-incubator/devworkspace-api/issues/32[corresponding issue].
-=======
-
-For a deeper understanding of adding events, go to the GitHub issue on the following topic:
```

Comment:
I think I would leave "
For a deeper understanding of adding events" in front of the sentence

Reply:
full stop is missing after link